### DISCUSSION TITLE
Powering the lamp

### DISCUSSION THREAD

**Discussion by Beatrix-droid** [martinwoodward/octolamp#12] [Q&A] [answered]
Hi first of all thank you for the project, and all the work you've put in it!
This is my very first electronics project, I am a junior software dev and am looking forward to making this lamp and adding it to my work from home setup!
I had a question about powering the lamp:
can I power the lamp by using a usb mini to usb cable? I'd connect the  usb mini to the d1 chip and plug the other end of the cable into my computer/a usb hub?
Many thanks!

**Comment by Likqez** [comment]
Hello, i don't think that would be agreat idea. 100 LEDs require around 20 watts of power. The USB 3.0 connector is rated for a maxium of 4.5Watts  (USB-C for 15 Watts). It probalby the best option to use an external 5V power supply.

**Comment by martinwoodward** [comment, marked as answer]
Hey @Beatrix-droid - that set-up works fine on my machine. The default WLED config limits the current draw for the device to 850mA so I haven't had any problems so far.  That said, if you want to crank the brightness up past the defaults then probably good to listen to Likqez's advice.  But also note that if you are using this on camera in a stream etc you want the brightness low to show up on screen
